crypto/objects/obj_dat.h

```diff
@@ -268,924 +268,924 @@ static const unsigned char so[8487] = {
     0x2B,0x06,0x01,0x05,0x05,0x07,0x00,0x10,       /* [ 2075] OBJ_id_mod_cmp2000 */
     0x2B,0x06,0x01,0x05,0x05,0x07,0x01,0x02,       /* [ 2083] OBJ_biometricInfo */
     0x2B,0x06,0x01,0x05,0x05,0x07,0x01,0x03,       /* [ 2091] OBJ_qcStatements */
-    0x2B,0x06,0x01,0x05,0x05,0x07,0x01,0x04,       /* [ 2099] OBJ_ac_auditEntity */
```

Comment:
Can you explain this huge change in Object ids?

Reply:
I am not sure what happened here. I will try regenerating this file and see what happens.

Reply:
It generated the exact same file.

Reply:
Were you looking for more information? I can dig deeper, but I think it's just a quirk of the script or git diff algorithm. For what it's worth, if you are concerned about me maliciously editing this file (fair enough), there is a CI/CD job that will check if my PR has any diff from the last commit after running all the code generation scripts.

Reply:
Can I mark this as resolved?

Reply:
After rebasing on `master` to resolve the merge conflict, it still has this large diff. I think this is just a consequence of adding that alias line. I even tried adding it to the end of `objects.txt` to see if that made a difference and the resulting file is exactly the same.

Reply:
Could you as an experiment edit the obj_mac.num file and switch the auditIdentity and auditEntity lines and numbers? I.e. set auditIdentity to the old value and auditEntity to the new one?

Reply:
To make sure I understood what you meant, here is the diff of what I did:

```patch
diff --git a/crypto/objects/obj_mac.num b/crypto/objects/obj_mac.num
index 6d8cd6262d..572e02257a 100644
--- a/crypto/objects/obj_mac.num
+++ b/crypto/objects/obj_mac.num
@@ -284,7 +284,7 @@ id_mod_dvcs         283
 id_mod_cmp2000         284
 biometricInfo          285
 qcStatements           286
-ac_auditEntity         287
+ac_auditIdentity               287
 ac_targeting           288
 aaControls             289
 sbgp_ipAddrBlock               290
@@ -1320,4 +1320,4 @@ associated_information            1319
 id_ct_rpkiSignedPrefixList             1320
 id_on_hardwareModuleName               1321
 id_kp_wisun_fan_device         1322
-ac_auditIdentity               1323
+ac_auditEntity         1323
```

After truncating further changes in the file, this is what it looks like, compared to `master`. What you would have expected from my original change: a small diff in `obj_dat.h`.

```patch
diff --git a/crypto/objects/obj_dat.h b/crypto/objects/obj_dat.h
index d86f5f83ad..4c61e964fd 100644
--- a/crypto/objects/obj_dat.h
+++ b/crypto/objects/obj_dat.h
@@ -268,7 +268,7 @@ static const unsigned char so[8504] = {
     0x2B,0x06,0x01,0x05,0x05,0x07,0x00,0x10,       /* [ 2075] OBJ_id_mod_cmp2000 */
     0x2B,0x06,0x01,0x05,0x05,0x07,0x01,0x02,       /* [ 2083] OBJ_biometricInfo */
     0x2B,0x06,0x01,0x05,0x05,0x07,0x01,0x03,       /* [ 2091] OBJ_qcStatements */
-    0x2B,0x06,0x01,0x05,0x05,0x07,0x01,0x04,       /* [ 2099] OBJ_ac_auditEntity */
+    0x2B,0x06,0x01,0x05,0x05,0x07,0x01,0x04,       /* [ 2099] OBJ_ac_auditIdentity */
     0x2B,0x06,0x01,0x05,0x05,0x07,0x01,0x05,       /* [ 2107] OBJ_ac_targeting */
     0x2B,0x06,0x01,0x05,0x05,0x07,0x01,0x06,       /* [ 2115] OBJ_aaControls */
     0x2B,0x06,0x01,0x05,0x05,0x07,0x01,0x07,       /* [ 2123] OBJ_sbgp_ipAddrBlock */
@@ -1187,7 +1187,7 @@ static const unsigned char so[8504] = {
     0x2B,0x06,0x01,0x04,0x01,0x82,0xE4,0x25,0x01,  /* [ 8494] OBJ_id_kp_wisun_fan_device */
 };
```

Reply:
I mean do this plus add the compat alias to objects.txt - this should minimize the changes, shouldn't it?

Reply:
It seems like that worked. Does this mean that it is okay that the definition of `NID_ac_auditEntity` changed?

Reply:
Yeah, IMO OK.